include/odp/api/spec/schedule_types.h

```diff
@@ -214,6 +214,27 @@ typedef struct odp_schedule_capability_t {
 
 } odp_schedule_capability_t;
 
+/**
+ * Schedule configuration
+ */
+typedef struct odp_schedule_config_t {
+	/** Maximum number of schedule queues to be supported.
+	 * Application configures the maximum number of schedule queues to be
+	 * supported by the implementation.
+	 * @see odp_queue_capability_t
+	 */
+	uint32_t queue_count;
```

Comment:
num_queues instead of count

... configures maximum number of scheduled queues required by the application. 

In general, "to be supported" or "to be supported by implementation" should be changed to "required" or "required by application".

Reply:
ack